### DISCUSSION TITLE
Anyway to fully delete an account?

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#23954] [Q&A]
I noticed that you are not able to full delete an account. 

I want to completely delete an account from my instance.

The issue I am having is you can only suspend the account so it still shows up in the admin accounts section. The username is forever taken and the email associated to that account can never again register. 

i want to completely remove it. 

anyway?

**Comment by Lastorder-DC** [comment]
No. you can't remove account and use same account name again. Using handle(`@id@domain`) again will cause lots of problem about federation(remote server might still have old toots from same handle), so Account `Deletion` is not possible.

**Reply by ghost** [reply]
What happens if i go into the database and forcefully delete the username

**Reply by Lastorder-DC** [reply]
That ~~will~~ might break federation since other servers ~~will~~ can reject your username If they have old ID's data, making Impossible to reach other server. DO NOT DO THAT.

**Reply by ghost** [reply]
This is terrible news. 

Why wouldn't the servers check and purge the old IS. 

Deleted accounts takes up space in the accounts page and i've deleted many.

Can you fix this in the future? it would be great!

**Reply by Lastorder-DC** [reply]
Deleted accounts does not take any space. any toot/media/etc will be removed after 30 days.(standard permanently suspended account behavior)

while you can manually delete deleted account's data, you don't need to do that becuse It will be automatically deleted.
